benchmark/kostya/yyjson.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include "kostya.h"
+
+namespace kostya {
+
+class Yyjson {
+public:
+  simdjson_really_inline const std::vector<my_point> &Result() { return container; }
+  simdjson_really_inline size_t ItemCount() { return container.size(); }
+
+private:
+  ondemand::parser parser{};
+  std::vector<my_point> container{};
+
+  simdjson_really_inline double get_double(yyjson_val *obj, std::string_view key) {
+    yyjson_val *val = yyjson_obj_getn(obj, key.data(), key.length());
+    return yyjson_get_real(val);
```

Comment:
I do not think that's quite correct. Here is how get_real is implemented:

```
double yyjson_get_real(yyjson_val *val) {
    return yyjson_is_real(val) ? unsafe_yyjson_get_real(val) : 0.0;
}
```

So you want to do...

```C++
if(yyjson_is_real(val)) {
  return unsafe_yyjson_get_real(val);
} else if(yyjson_is_uint(val)) {
  return double(unsafe_yyjson_get_uint(val));
} else if(yyjson_is_sint(val)) {
  return double(unsafe_yyjson_get_sint(val));
} else {
????
}
```

Reply:
This is how I handle it in py_yyjson: https://github.com/TkTech/py_yyjson/blob/main/yyjson/binding.c#L51

Reply:
Fixed!